modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/surface/Surface.js

```diff
@@ -90,27 +89,33 @@ class Surface extends Disposable {
 	 * @return {Element}
 	 */
 	addContent(screenId, opt_content) {
-		var child = this.defaultChild;
+		const fragment =
+			typeof opt_content === 'string'
```

Comment:
If you can be bothered, now might be a nice time to rename `opt_content` to get rid of the underscore and switch to camel case (bonus points if you get rid of the "opt" abbreviation too).